manifests/kustomize/base/pipeline/ml-pipeline-apiserver-deployment.yaml

```diff
@@ -115,6 +115,10 @@ spec:
             secretKeyRef:
               name: mlpipeline-minio-artifact
               key: secretkey
+        - name: V2_DRIVER_IMAGE
+          value: ghcr.io/kubeflow/kfp-driver:2.4.0
+        - name: V2_LAUNCHER_IMAGE
+          value: ghcr.io/kubeflow/kfp-launcher:2.4.0
```

Comment:
I'm not sure I'm entirely following this change.

Can't we set `latest` here and replace it with the version in releases? By having `latest` here, contributors would use the latest images when deploying from the `master` branch, which makes more sense in development.

Otherwise, I think you meant `2.4.1`.

```suggestion
        - name: V2_DRIVER_IMAGE
          value: ghcr.io/kubeflow/kfp-driver:2.4.1
        - name: V2_LAUNCHER_IMAGE
          value: ghcr.io/kubeflow/kfp-launcher:2.4.1
```

Reply:
I did however correct the patch version as per your suggestion